app/containers/message/Attachments.tsx

```diff
@@ -51,6 +52,17 @@ const Attachments = React.memo(
 			if (file.actions && file.actions.length > 0) {
 				return <AttachedActions attachment={file} theme={theme} />;
 			}
+			if (file.collapsed)
```

Comment:
We should conditionally show this component if `file.title`, like they do on the web https://github.com/RocketChat/Rocket.Chat/blob/7632f12cfcc7ed8ee8f843587fdff63b101cc765/client/components/Message/Attachments/DefaultAttachment.tsx#L57

Reply:
It went bad :(